BlackJackets-ui/src/pages/GuestView.jsx

```diff
@@ -6,7 +6,8 @@ import { useParams } from "react-router-dom";
 function GuestView() {
   const [venue, setVenue] = useState([]);
   const [gigs, setGigs] = useState([]);
-  const {id} = useParams();
+
```

Comment:
Need to add: `const navigate = useNavigate();`

Reply:
done